docker-compose.developer.full.yaml

```diff
@@ -172,8 +172,6 @@ services:
       - CRAY_VAULT_JWT_FILE=configs/token
       - CRAY_VAULT_ROLE_FILE=configs/namespace
       - CRAY_VAULT_AUTH_PATH=auth/token/create
-      - HTTPS_CERT=configs/rts.crt
```

Comment:
why are these removed?

Reply:
I took them out when I thought they weren't needed.   I added them back in for the unit test yaml file -- I will put these back in.

Reply:
can you look at dynamically generating: https://github.com/Cray-HPE/hms-tools/blob/master/hwval/gen-certs.sh

Reply:
Will try.

Reply:
OK, made the .crt and .key files "ephemeral".